server/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "source .env && ENV=prod nodemon -r esm app.js",
-    "dev": "source .env && ENV=dev nodemon -r esm app.js",
+    "start": "ENV=prod node -r esm app.js",
```

Comment:
"start" script no longer uses nodemon. Is this correct?

Reply:
Yes, this is for the purpose when deploy to Gcloud App Engine